config/vendor-api-v1.yml

```diff
@@ -591,8 +592,16 @@ components:
         uk_residency_status:
           type: string
           maxLength: 256
-          description: The candidate’s UK residency status indicating their right to work and study in the UK
+          description: The candidate’s UK residency status indicating their right to work and study in the UK. Possible values include 'UK Citizen', 'Irish Citizen', 'Candidate needs to apply for permission to work and study in the UK' and 'Candidate does not know'. The candidate can also provide details as free text for example "Settled status".
           example: UK Citizen
+        fee_payer:
+          type: string
+          maxLength: 2
+          description: Provisional fee payer status based on a candidate's nationality, right to work or study in the UK and domicile.
```

Comment:
This content looks good to me but might need to double check with Oliver just incase

Reply:
Oli suggested replacing "right to work or study in the UK" with "residency status" here :)